src/plugins/template/tests/functional/skip_tests_config.cpp

```diff
@@ -121,6 +121,8 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*eltwiseOpType=Mod_secondaryInputType=PARAMETER_opType=VECTOR_NetType=(f16|f32).*)",
         // Interpreter backend doesn't implement evaluate method for OP Multiply (by GroupNormalizationDecomposition)
         R"(.*ReferenceGroupNormalization.*_f64*)",
+        // CVS-131733
+        R"(.*smoke_BehaviorTests/InferRequestPropertiesTest.ReusableCPUStreamsExecutor.*)",
```

Comment:
the test body does not contains skip condition. So, I suppose current statement does not take any effect

Reply:
The skip is in InferRequestPropertiesTest::SetUp. 
But the problem is this test fails for CPU now. So probably the root cause is not in template plugin